maintainers/github-teams.json

```diff
@@ -529,7 +524,6 @@
     "members": {
       "K900": 386765,
       "Ma27": 6025220,
-      "NeQuissimus": 628342,
```

Comment:
As the maintainer for the linux-kernel team, @alyssais, please implement the removal of `NeQuissimus` from the team.

Reply:
Now done myself in https://github.com/NixOS/nixpkgs/pull/471116